Formula/i/ifopt.rb

```diff
@@ -0,0 +1,26 @@
+class Ifopt < Formula
+  desc "Light-weight C++ Interface to Nonlinear Programming Solvers"
+  homepage "https://wiki.ros.org/ifopt"
+  url "https://github.com/ethz-adrl/ifopt/archive/refs/tags/2.1.4.tar.gz"
+  sha256 "da38f91a282f3ed305db163954c37d999b6e95f5d2c913a63bae3fef9ffb3a37"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "eigen"
+  depends_on "ipopt"
+
+  def install
+    system "cmake", "-B", "build", "-DCMAKE_POLICY_VERSION_MINIMUM=3.5", *std_cmake_args
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", "-DCMAKE_POLICY_VERSION_MINIMUM=3.5", *std_cmake_args
```
Let's follow the style used by other formulae.

Reply:
My bad, not sure how I missed that.